FileAPI/url/cross-global-revoke.sub.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<meta charset="utf-8">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+async_test(t => {
```

Comment:
Might it be more readable to use a "straight-line" async arrow function here instead, and `await` with one-shot handlers, e.g. `const first_message = await new Promise(resolve => self.onmessage = e => e.source == frame.contentWindow && resolve(e.data));` `assert_equals(first_message, 'ready');` and then later `const second_message = await new Promise(resolve => self.onmessage = e => e.source == frame.contentWindow && resolve(e.data));` `assert_equals(second_message, 'revoked')`, rather than `step_func` and nested sync handler scopes?

Reply:
👍 Those are good reasons. Thanks for explaining and dispelling my ignorance!